src/digest.rs

```diff
@@ -279,8 +279,7 @@ pub struct Algorithm {
     /// The length of the length in the padding.
     len_len: usize,
 
-    block_data_order: unsafe extern fn(state: *mut u64, data: *const u8,
-                                       num: c::size_t),
+    block_data_order: unsafe extern fn(state: *mut u64, data: *const u8, num: c::size_t),
```

Comment:
Please undo this change. (If you are using rustfmt, you need to set it to use an ~80 character limit.